src/js/yeldaWebchatInjector.js

```diff
@@ -949,6 +965,48 @@ class YeldaChat {
       webchatFrame.contentWindow.postMessage({ event: 'sendUserMessage', data: message })
     }
   }
+
+  /**
+   * Listen leave viewport event handling
+   * @param {Boolean} [remove=false]
+   */
+  listenLeaveViewport(remove = false) {
```

Comment:
Even better than `remove`: `shouldRemoveEvent`. With the name itself we can understand that it is a boolean, and what it will be used for.